### DISCUSSION TITLE
Error running process: extensions\sd-webui-controlnet\scripts\controlnet.py

### DISCUSSION THREAD

**Discussion by Bhagathxoxo** [Mikubill/sd-webui-controlnet#450] [Q&A]
I'm using control net from two days, then I findout that I can use two controlnet models at same time, then in the setting > Controlnet> Multi ControlNet: Max models amount: I changed it from 1 to  2. After reloading the webUI, When I use the controlnet it showing the error below and rendered the normal stable diffusion output. I again changed back the Multi ControlNet to 1 and tried it, and same normal stable diffusion render without control net.
I reinstalled the extension, not working.
I reinstalled the Automatic1111, not working.
I installed everything from again, still no use. I tried updating the extension or webui and no use.
**Please Help. This is the error message:**

Loading model: control_sd15_canny [fef5e48e]
Loaded state_dict from [D:\AI\stable-diffusion-webui\extensions\sd-webui-controlnet\models\control_sd15_canny.pth]
Error running process: D:\AI\stable-diffusion-webui\extensions\sd-webui-controlnet\scripts\controlnet.py
Traceback (most recent call last):
  File "D:\AI\stable-diffusion-webui\modules\scripts.py", line 386, in process
    script.process(p, *script_args)
  File "D:\AI\stable-diffusion-webui\extensions\sd-webui-controlnet\scripts\controlnet.py", line 612, in process
    else self.build_control_model(p, unet, model, lowvram)
  File "D:\AI\stable-diffusion-webui\extensions\sd-webui-controlnet\scripts\controlnet.py", line 471, in build_control_model
    network = network_module(
  File "D:\AI\stable-diffusion-webui\extensions\sd-webui-controlnet\scripts\cldm.py", line 63, in __init__
    config = OmegaConf.load(config_path)
  File "D:\AI\stable-diffusion-webui\venv\lib\site-packages\omegaconf\omegaconf.py", line 187, in load
    with io.open(os.path.abspath(file_), "r", encoding="utf-8") as f:
FileNotFoundError: [Errno 2] No such file or directory: 'D:\\AI\\stable-diffusion-webui\\models\\cldm_v15.yaml'

**Comment by Mikubill** [comment]
Fixed in https://github.com/Mikubill/sd-webui-controlnet/commit/9b19c63e9872e3dc059c07a592bbe83250e8ce79 - update extension and it should works

**Comment by namdoyle** [comment]
I have same error, but after using --lowvram argument (webui_user.bat) to start the app, the problem is gone.

**Comment by Ramoyald** [comment]
your fix is pretty clear in this error message, you are missing a pretty important file

FileNotFoundError: [Errno 2] No such file or directory: 'D:\AI\stable-diffusion-webui\models\cldm_v15.yaml'
